### DISCUSSION TITLE
Error - Unable yo install OpenRoad

### DISCUSSION THREAD

**Discussion by mehta-nitin** [The-OpenROAD-Project/OpenROAD#4038] [Q&A]
I have tried to build the file again and again and tried troubleshooting but couldn't find the solution.
Please help me with the same.
The error ca be seen in the image below.
![1](https://github.com/The-OpenROAD-Project/OpenROAD/assets/62429762/a297f803-b2c0-471a-8cdf-64e08515bcda)

**Comment by vvbandeira** [comment]
Hi @mehta-nitin,
What you describe is an issue, not a discussion. We need more info on your system, environment, and the steps you have tried / how to reproduce -- please use the link below to file a GitHub Issue and follow our template instructions so we can better help you.
https://github.com/The-OpenROAD-Project/OpenROAD/issues/new/choose
